crates/bevy_sprite/src/mesh2d/color_material.wgsl

```diff
@@ -26,12 +26,11 @@ struct FragmentInput {
 @fragment
 fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
     var output_color: vec4<f32> = material.color;
+    #ifdef VERTEX_COLORS
+    output_color = output_color * in.color;
+    #endif
     if ((material.flags & COLOR_MATERIAL_FLAGS_TEXTURE_BIT) != 0u) {
-#ifdef VERTEX_COLORS
-        output_color = output_color * textureSample(texture, texture_sampler, in.uv) * in.color;
-#else
         output_color = output_color * textureSample(texture, texture_sampler, in.uv);
-#endif
     }
     return output_color;
```

Comment:
I think you should keep the `#ifdef` and `#endif` start of line for consistency with all other shader code.

Reply:
Good point, pushed an update.